tests/test_user_model.py

```diff
@@ -0,0 +1,60 @@
+"""User model tests."""
+
+from unittest import TestCase
+from sqlalchemy import exc
+
+from models import db, User, Review
+
+from app import app
+
+app.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql:///covidtest'
+
+
+db.create_all()
```

Comment:
@azaria-dedmon this should be in the tests setup.

Reply:
Should I be listing these small changes under the PR description i.e. moving testing files to their own folder, re-arranging code, etc?

Reply:
No it is not necessary.